storio-sqlitedb/src/main/java/com/pushtorefresh/storio/sqlitedb/operation/delete/DeleteResults.java

```diff
@@ -0,0 +1,90 @@
+package com.pushtorefresh.storio.sqlitedb.operation.delete;
+
+import android.support.annotation.NonNull;
+
+import java.util.Collections;
+import java.util.Map;
+
+import static com.pushtorefresh.storio.util.Checks.checkNotNull;
+
+/**
+ * Immutable container for results of Delete Operation
+ * <p>
+ * Instances of this class are Immutable
+ *
+ * @param <T> type of objects
+ */
+public class DeleteResults<T> {
+
+    @NonNull
+    private final Map<T, DeleteResult> results;
+
+    private DeleteResults(@NonNull Map<T, DeleteResult> results) {
+        checkNotNull(results, "Please specify Delete Operation results");
+        this.results = Collections.unmodifiableMap(results);
+    }
+
+    /**
+     * Creates new instance of immutable container for results of Delete Operation
+     *
+     * @param results map with results of Delete Operation
+     * @param <T>     type of objects
+     * @return new instance of {@link DeleteResults}
+     */
+    @NonNull
+    public static <T> DeleteResults<T> newInstance(@NonNull Map<T, DeleteResult> results) {
+        return new DeleteResults<>(results);
+    }
+
+    /**
+     * Returns immutable Map of pairs (object, deleteResult)
+     *
+     * @return immutable Map of pairs (object, deleteResult)
+     */
+    @NonNull
+    public Map<T, DeleteResult> results() {
+        return results;
+    }
+
+    /**
+     * Checks whether particular object was deleted
+     *
+     * @param object object to check
+     * @return true if object was deleted, false otherwise
+     */
+    public boolean wasDeleted(@NonNull T object) {
+        return results.containsKey(object);
+    }
+
+    /**
+     * Checks whether particular object was NOT deleted
+     *
+     * @param object object to check
+     * @return true if object was NOT deleter, false if it was deleted
```

Comment:
Hm, good catch, yes, I think it's better to use `<code>true</code>` than just `true`, and I see typo in `NOT deleter` should be -> `NOT deleted`

I'll create issues for that